src/design-tokens.ts

```diff
@@ -3,10 +3,28 @@
 
 import {create} from './utilities/design-tokens/create';
 
+/**
+ * Developer note:
+ *
+ * There are some tokens defined in this file that make use of `--fake-vscode-token`. This is
+ * done when a toolkit token should be added to the tokenMappings map (and subsequently altered
+ * in the applyTheme function) but does not have a corresponding VS Code token that can be used.
+ *
+ * An example is buttonIconHoverBackground token which does not have a corresponding VS Code token
+ * at this time (it's a hardcoded value in VS Code), but needs to be adjusted to be transparent when a
```

Comment:
Should this be an issue filed on /vscode? If it's a simple enough change I can add it to my list

Reply:
I wouldn't be against that! Would certainly simplify the toolkit theming code

Reply:
The big request is that this token should either be set to transparent or null (not exist) in high contrast themes

Reply:
Otherwise, its value seems to be `rgba(90, 93, 94, 0.31)` across all themes?

Reply:
As I think about it, this could actually be a really nice opportunity to customize things a bit because I think this value ends up resulting in a less than ideal contrast ratio in light themes

Reply:
<img width="130" alt="Screen Shot 2022-02-03 at 2 50 17 PM" src="https://user-images.githubusercontent.com/39639992/152442471-552c1409-74a3-46c7-a1c2-a982fde1d51a.png">